src/components/shared/ArrowPanel.css

```diff
@@ -33,8 +33,11 @@
 
 .arrowPanel:not(.open) {
   pointer-events: none;
-  transition: opacity 200ms ease-out;
   opacity: 0;
+  /* visibility: hidden hides the panel from the accessibility tree when it's closed */
+  visibility: hidden;
+  /* The visibility is set to hidden after the panel closing transition. */
+  transition: opacity 200ms ease-out, visibility 0s 200ms;
```

Comment:
That's a cool trick.

Reply:
We could do just `visibility 200ms` due to how visibility interpolation works, but I found that doing it this way was more understandable to a reader that's not proficient with CSS.